sections/disciplines.md

```diff
@@ -0,0 +1,45 @@
+Disciplines
+=========
+
+Disciplines currently must be created and assigned through the UI.
+
+Endpoints:
+
+- [Get Disciplines](#get-disciplines)
+- [Get a Discipline](#get-a-discipline)
+
+
+Get Disciplines
+-------------
+
+* `GET /api/v1/discipline` will return a [paginated list](/README.md#pagination) of Disciplines.
+
+###### Example JSON Response
+
+```json
+[
+  {
```

Comment:
When it's paginated, there's a `paging` key and data. It'd be good to document that here.